internal/testFixtures.ts

```diff
@@ -17,8 +17,8 @@ export const exampleEnum: FoxgloveEnumSchema = {
   parentSchemaName: "ExampleMessage",
   description: "An example enum",
   values: [
-    { name: "A", value: 1, description: "Value A" },
-    { name: "B", value: 2, description: "Value B" },
+    { name: "A", value: 0, description: "Value A" },
```

Comment:
Is it worth putting a comment here that these need to start at 0 for omgidl enum support?

Reply:
I think it's fine without since the assertion will catch it if this were to change